docs/samples/calling/style.css

```diff
@@ -254,18 +254,13 @@ input[type=email]:focus {
 
 /* Container - Docs / Streams Fixed */
 .container {
-  display: grid;
-  position: relative;
-  max-width: 1400px;
-  margin: 1rem auto;
-  grid-gap: 2rem;
-  grid-template-columns: 1fr 400px;
-  padding-bottom: 1rem;
+  overflow: auto;
+  height: 100vh;
 }
 
 .docs {
-  overflow: auto;
-  height: 98vh;
+  max-width: 1400px;
```

Comment:
can we convert this `px` value to ``rem value here ?
replace to
`max-width: 87.5rem`